### PR TITLE
Updatelogo

### LINKED ISSUE #79: Update Logo
## Background

The logo has changed for the 2020 competition.

Assigned to @waiholiu

## Tasks

- [x] Provide new bannerlogo.png
- [x] Provide new logo for favicon.
  - Just a square logo, I can generate the favicon (Cassar)
- [ ] Replace all instances of the logo.

## Notes

- Existing images can be found in `app/assets/images`
- Use the [favicon genenerator ](https://realfavicongenerator.net/favicon/ruby_on_rails#.Xv0Of_IRV24)
- The mailer layout will also need to be updated.

### PR DESCRIPTION
**Pull request by waiholiu**
Fixes #79

### CODE REVIEW

File: app/views/application/_favicon.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<link rel="apple-touch-icon" sizes="180x180" href="<%= asset_path 'favicon/apple-touch-icon.png' %>">
```
Comment (cassar) [inline app/views/application/_favicon.html.erb:1]: We may not need this as it's the html version of `app/views/application/_favicon.html.haml`.

If there are any updates here we can also just delete the old one and keep this one.

**Comment by cassar** [comment]
Just gave this a go, favicon seems to be working 👍

File: app/views/application/_favicon.html.erb (hunk 1 above)
Comment (cassar) [reply app/views/application/_favicon.html.erb]: Ahh apologies, it was the whole file I was commenting on, not that particular line, 😅 

Have pushed a fixup that resolves this.

**Comment by waiholiu** [comment]
hey apologies for that, looks like i had forgotten to push the commit with the logos - anyway should be sorted now - i also did a merge to get it to the latest main branch.

**Pull request merged**